src/firmware_new/boards/frankie_v1/io/drivetrain_unit.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "firmware_new/boards/frankie_v1/io/allegro_a3931_motor_driver.h"
+
+// TODO: better name for this file/"class"?
```

Comment:
`drivetrain_unit` is really intuitive imo, not sure what would be better

Reply:
Whoops, I had roughly come to the same conclusion, meant to remove this. Removed now.